### ISSUE TITLE
Launch in Google Colab button

### ISSUE THREAD

**Issue by thesamovar** [executablebooks/mystmd#1174]
## Proposal

It would be cool to have a launch in Google Colab button. Lots of people have pro accounts that make this a nice option for them compared to Binder, plus persistence, etc.

**Comment by giswqs** [comment]
I second this. Jupyter Book already has this feature. It would be nice if MyST can support it as well.

**Comment by chillerb** [comment]
Yes, I would also love to have this feature back! Google Colab is still nice for co-operative programming with other people and allowing to store changes/edits in the User's Google Drive.

Maybe there could be some configuration of pre-defined external computing interfaces?

<img width="509" height="401" alt="Image" src="https://github.com/user-attachments/assets/11fee248-fd58-4674-868a-cad7e53ce9c0" />

On a side note, as a first time jupyter book v2 user, I was a little confused that the mybinder link just links to the mybinder homepage, requiring manual copy-pasting of the link into the url field 😅

**Comment by choldgraf** [comment]
Indeed - we have this issue tracking the ability to define a default launch binder:

- #2070
